examples/sqlx_mysql/src/main.rs

```diff
@@ -4,6 +4,7 @@ use rust_decimal::Decimal;
 use sea_query::{ColumnDef, Expr, Func, Iden, MysqlQueryBuilder, OnConflict, Order, Query, Table};
 use sea_query_binder::SqlxBinder;
 use sqlx::{types::chrono::NaiveDateTime, MySqlPool, Row};
+use std::ops::DerefMut;
```

Comment:
can we remove this

Reply:
We import `std::ops::DerefMut` into scope because we call the `.deref_mut()` method.